### PR TITLE
Added openshift_rhv role for rhv provider

### PR DESCRIPTION
**Pull request by cwilkers**
Trying to break up the extra-extra-large PR into smaller chunks. Here is just the openshift_rhv role from: 
#8948 

@sdodson @cooktheryan

### CODE REVIEW

**Comment by cwilkers** [comment]
bot, retest this please

File: roles/openshift_rhv/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+For default values, see [`defaults/main.yaml`](defaults/main.yaml).
+
+For documentation on virtual machine profile options, see the [oVirt Ansible VM-Infra Documentation](https://github.com/oVirt/ovirt-ansible-vm-infra)
+
+- `openshift_rhv_vm_profile`: Dictionary of dictionaries providing common VM parameters for virtual machine creation.
+- `openshift_rhv_vm_manifest`: List of dictionaries specifying node base name, count, and which of the above profiles to apply. The default creates three master nodes, three infrastructure nodes, three application nodes, and a load balancer.
+
+```
+openshift_rhv_vm_manifest:
```
Comment (mazzystr) [inline roles/openshift_rhv/README.md]: Element indentation doesn't follow openshift-ansible standard

File: roles/openshift_rhv/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+Example Playbook
+----------------
+
+```
+- name: Create dns nsupdate or hosts files for admin use
+  hosts: localhost
+  tasks:
+    - import_role:
```
Comment (mazzystr) [inline roles/openshift_rhv/README.md]: Element indentation doesn't follow openshift-ansible standard

File: roles/openshift_rhv/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+# oVirt VM Profiles
+openshift_rhv_vm_profile:
+  master:
+    cluster: "{{ openshift_rhv_cluster }}"
+    template: "{{ ovirt_template_name }}"
+    memory: 16GiB
+    cores: 2
+    high_availability: true
+    disks:
```
Comment (mazzystr) [inline roles/openshift_rhv/defaults/main.yml]: Element indentation doesn't follow openshift-ansible standard

File: roles/openshift_rhv/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        interface: virtio
+    state: running
+  node:
+    cluster: "{{ openshift_rhv_cluster }}"
+    template: "{{ ovirt_template_name }}"
+    memory: 8GiB
+    cores: 2
+    high_availability: true
+    disks:
+      - size: 15GiB
```
Comment (mazzystr) [inline roles/openshift_rhv/defaults/main.yml]: Element indentation doesn't follow openshift-ansible standard

**Comment by cwilkers** [comment]
Closing due to change in handling oVirt roles

**Pull request closed without merging**